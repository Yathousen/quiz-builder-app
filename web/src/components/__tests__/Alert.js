import React from 'react';
import renderer from 'react-test-renderer';
import { Alert } from '../';

it('Alert Shows', () => {
    expect(renderer
      .create(<Alert show={true} text='Alert' />)
      .toJSON()).toMatchSnapshot();
});

it("Alert Doesn't show", () => {
  expect(renderer
    .create(<Alert show={false} text='Alert' />)
    .toJSON()).toMatchSnapshot();
});
