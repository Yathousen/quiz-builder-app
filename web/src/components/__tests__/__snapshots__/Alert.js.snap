// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert Doesn't show 1`] = `
<div
  className="hidden w-full text-white rounded mt-2 bg-gray-500"
>
  <div
    className="container flex items-center justify-between px-6 py-4 mx-auto"
  >
    <div
      className="flex"
    >
      <svg
        className="w-6 h-6 fill-current"
        viewBox="0 0 40 40"
      >
        <path
          d="M20 3.36667C10.8167 3.36667 3.3667 10.8167 3.3667 20C3.3667 29.1833 10.8167 36.6333 20 36.6333C29.1834 36.6333 36.6334 29.1833 36.6334 20C36.6334 10.8167 29.1834 3.36667 20 3.36667ZM19.1334 33.3333V22.9H13.3334L21.6667 6.66667V17.1H27.25L19.1334 33.3333Z"
        />
      </svg>
      <p
        className="mx-3"
      >
        Alert
      </p>
    </div>
  </div>
</div>
`;

exports[`Alert Shows 1`] = `
<div
  className=" w-full text-white rounded mt-2 bg-gray-500"
>
  <div
    className="container flex items-center justify-between px-6 py-4 mx-auto"
  >
    <div
      className="flex"
    >
      <svg
        className="w-6 h-6 fill-current"
        viewBox="0 0 40 40"
      >
        <path
          d="M20 3.36667C10.8167 3.36667 3.3667 10.8167 3.3667 20C3.3667 29.1833 10.8167 36.6333 20 36.6333C29.1834 36.6333 36.6334 29.1833 36.6334 20C36.6334 10.8167 29.1834 3.36667 20 3.36667ZM19.1334 33.3333V22.9H13.3334L21.6667 6.66667V17.1H27.25L19.1334 33.3333Z"
        />
      </svg>
      <p
        className="mx-3"
      >
        Alert
      </p>
    </div>
  </div>
</div>
`;
